#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_update_external_links.dpatch by David Gil <dgil@telefonica.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fixed location of signatures for Nessus (although the previous link
## DP: works) and for ICAT (it is now the NVD - National Vulnerability
## DP: Database)

@DPATCH@
diff -urNad acidbase~/base_conf.php.dist acidbase/base_conf.php.dist
--- acidbase~/base_conf.php.dist	2010-07-03 01:41:42.000000000 -0400
+++ acidbase/base_conf.php.dist	2010-07-03 01:45:45.000000000 -0400
@@ -357,6 +357,8 @@
 
 /* Whois query */
 $external_whois_link = 'http://www.dnsstuff.com/tools/whois.ch?ip=';
+/* Alternative query */
+//  $external_whois_link = 'http://www.samspade.org/t/ipwhois?a=';
 
 
 /* Local whois */
@@ -385,6 +387,8 @@
 
 /* DNS query */
 $external_dns_link = 'http://www.dnsstuff.com/tools/ptr.ch?ip=';
+/* Alternative query */
+//  $external_dns_link = 'http://www.samspade.org/t/dns?a=';
 
 /* SamSpade 'all' query */
 $external_all_link = 'http://www.whois.sc/';
@@ -396,11 +400,10 @@
 
 /* Signature references */
 $external_sig_link = array('bugtraq'   => array('http://www.securityfocus.com/bid/', ''),
-                           'snort'     => array('http://www.snort.org/search/sid/', ''),
+                           'snort'     => array('http://www.snortid.com/snortid.asp?QueryId=', ''),
                            'cve'       => array('http://cve.mitre.org/cgi-bin/cvename.cgi?name=', ''),
-                           'arachnids' => array('http://www.whitehats.com/info/ids', ''),
                            'mcafee'    => array('http://vil.nai.com/vil/content/v_', '.htm'),
-                           'icat'      => array('http://icat.nist.gov/icat.cfm?cvename=CAN-', ''),
+                           'icat'      => array('http://nvd.nist.gov/nvd.cfm?cvename=CAN-', ''),
                            'nessus'    => array('http://www.nessus.org/plugins/index.php?view=single&amp;id=', ''),
                            'url'       => array('http://', ''),
                            'local'     => array('signatures/', '.txt'),
